notebooks/textbook/Quantum_Neuron/Quantum_Neuron.ipynb

```diff
@@ -0,0 +1,717 @@
+    "\n",
+    "The below shows classical and quantum neuron, based on Figure 1 of the paper. At the end of this work, you can get to learn how to build a non-linear transformation from linear nature of quantum mechanics.\n",
+    "<div align=\"center\"><img src=\"img/classical_and_quantum_neuron.png\"/></div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Building Linear Combination Circuit\n",
+    "\n",
+    "Before we explain a quantum neuron, let us review a classical neuron. A classical neuron is a function that takes $n$ variables $x_1, \\cdots, x_n$ and maps them to the output value $a=\\sigma\\left(w_1 x_1+w_2 x_2+\\cdots+w_n x_n+b\\right)$ with $\\{w_i\\}$ and $b$ being the synaptic weights and bias, respectively (shown in (a)). The quantity $\\theta = w_1 x_1+w_2 x_2+\\cdots+w_n x_n+b$ is called the input signal to the neuron. The activation function $\\sigma(z)$ is a nonlinear function such as the step function and the sigmoid function.\n",
+    "\n",
+    "In order to mimic the classical neuron, first, we use inputs $\\left| x \\right> = \\left| x_1 ... x_n  \\right>$ as a control state and apply $R_y(2w_i)$ onto an ancilla qubit conditioned on the $i$-th qubit, followed by $R_y(2b)$ on the ancilla qubit. This ammounts to applying the $R_y(2\\theta)$ on the ancilla qubit conditioned on the input state $\\left| x \\right>$ (shown in (b) and below). (At the bottom of the below figure, we simplify these circuits). This process means the linear combination of the inputs $\\{x_i\\}$, the weights $\\{w_i\\}$, and bias $b$.\n",
+    "\n",
+    "<div align=\"center\"><img src=\"img/linear_combination_circuit.png\"/></div>\n",
+    "\n",
+    "We can calculate the effect of the circuit:\n",
+    "$$\n",
+    "\\begin{split}\n",
+    "    &\\left| x_1 ... x_n  \\right> \\otimes \\left| 0 \\right> \\\\\n",
+    "    &\\xrightarrow{1} \\left| x_1 ... x_n \\right> \\otimes Ry(2w_1 x_1) \\left| 0 \\right> \\\\\n",
+    "    &\\xrightarrow{2} \\left| x_1 ... x_n \\right> \\otimes Ry(2w_1 x_1 + 2w_n x_n) \\left| 0 \\right> \\\\\n",
+    "    &\\xrightarrow{3} \\left| x_1 ... x_n \\right> \\otimes Ry(2w_1 x_1 + 2w_n x_n + 2b) \\left| 0 \\right> \\\\\n",
+    "    &= \\left| x_1 ... x_n \\right> \\otimes Ry(2\\theta) \\left| 0 \\right> \\\\\n",
+    "\\end{split}\n",
+    "$$\n",
+    "where the two terms separated by tensor product represent, from left to right, inputs, and ancilla qubit(s), respectively."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Build Activation Function Circuit\n",
+    "\n",
+    "Secondly, after building a linear combination circuit, we build an activation function circuit. However it seems a unique idea that came out of nowhere, we show how to build an activation function circuit below.\n",
```

Comment:
> that came out of nowhere

In stating that this came out of nowhere, you should also include the motivation for including it so the reader can understand why it is introduced.

Reply:
> Secondly, after building a linear combination circuit, we build an activation function circuit. However it seems a unique idea that came out of nowhere, we show how to build an activation function circuit below.

is changed to

> Then, like in a classical neuron, the linear combination is followed by an activation function. we show how to build an activation function circuit below.